### DISCUSSION TITLE
심화프로젝트 최종 팀 회고

### DISCUSSION THREAD

**Discussion by sulyoungoh** [codeit-bootcamp-frontend/3-Weekly-Mission#349] [회고]
길었던 심화 프로젝트 끝이 다가오고 있어요!
정말 너무 너무 고생 많으셨습니다🙏

마지막 프로젝트인 만큼 아쉬움이 남을 수도, 혹은 후련할 수도 있을 것 같은데요
이번 최종 회고에서는 이를 편안하게 나누어볼 수 있는 자리가 되었으면해요!

너무 고생 많으셨습니다 🥰

마지막 팀 회고 후([회고 방식 상세보기](https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/discussions/346#discussion-6517270)), 이전처럼 댓글 남겨주세요🙏

마지막까지 파이팅💪

**Comment by ohddang** [comment]
# 3팀

## K 
- 모두 최선을 다한다.
- 거의 다 됐으니 이대로만 갑시다
- 일정에 맞춰서 기능 구현 함
- 디자이너 탈출에도 굴하지 않고 빠르게 디자인 적용

## P
- 디자이너 런해서 우리가 디자인 해야함
- 이제 진짜 시간이 얼마 남지 않았다.
- 테스트 코드를 많이 짜진 못했다.
- No Problem
- 체력이 슬슬 힘든게 느껴짐

## T
- 함께 디자인 꿈을 펼쳐보아요
- 세부 기능 구현 및 버그 수정
- 끝나고도 모두와 잘지내기
- 잘먹고 잘잡시다

**Comment by jinahp** [comment]
# 6팀
### K
- 서비스의 완성도에 다들 욕심이 있다.
- 다들 우리 팀이 정해둔 기간 내에 기능 구현 완성하려고 노력 많이 했다.
- 우리 팀의 좋은 점이 너무 많아서 하나만 꼽는게 쉽지 않은데,, 무리하지 않게 서로를 배려하는 모습이 너무 좋았습니다 ㅎㅎ
- 맡은 영역을 모두 책임감있게 해내고 거의 다 왔습니다.

### P
- 다들 현재 주어진 환경에서 열심히 하고 있고, 바로 생각 나는 문제점은 딱히 없다.
- 준용님이 깃허브에 각자 회고 적자고 하셨는데.....끝까지 하지 못했다.....반성합니다
- ㄴ 헉.. 그랬었나요?!... 죄송합니다... 🥹
- 코드 분해로 다른 컴포넌트에 있는 css를 읽지 못한다 -> 하지만 코드 분해가 아주 잘 이루어졌다는 긍정적인 신호 -> 각 컴포넌트에 맞는 해당 css 파일을 잘 분리해서 넣어주면 될듯해요

### T
- 힘을 내서 끝까지 유종의 미를 거두자
- 부지런하고 똑똑한 준용님을 본받자
- 6팀 만세
- 발표까지 잘 끝마치고 수료식 때 봬요 !!! ㅎㅎㅎ
- 밥 같이 뚝딱!

**Comment by DanteSnow** [comment]
# 5팀

# **👍🏻**Keep

---

- 칸반보드를 적극 활용하여 각 팀원들이 무슨 작업을 하는지, 어떤 작업을 해야 하는지를 한 눈에 볼 수 있어서 작업 일정 조율에 도움이 됐습니다.
- 작업을 할 때 모두가 게더타운에 상주하며 열정을 불태워서 외롭지 않고 재미있게 작업할 수 있었습니다.
- 변경 사항이 있으면 빠르게 공유해주셔서 감사했습니다.
- 열심히 하는 모습을 보면 나도 자극 받는다!
- 웬만하면 게더에서 모각코 하고 계셔서 의견 공유가 빠르게 될 수 있어서 좋았습니다.

## **🤔**Problem

---

- 기간을 생각하지 못하고 과하게 기획을 한 것 같아서 아쉽습니다. 일정에 맞춰 작업에 허덕이는 팀원 분들을 보면 가슴이 아픕니다.
- 소통이 부족해서 작업 일정에 살짝 영향을 미친 것이 아쉽습니다.
- 정해 놓은 대로 완성이 되지 않아 작업에 차질이 생겼던 것 같습니다.
- 수월한 작업을 위한 원활한 소통이 서로 잘 이루어지지 않는 것 같아 아쉽습니다.

## **🔥**Try

---

- 조금 더 디테일하게 전체 일정을 수립했다면 좋았을 것 같습니다.
- 기다리는 다음 작업자를 위해 빠른 작업과 상세한 작업 진행 상황 공유가 필요하다고 생각함
- 사소한 안건이라도 회의 때 나온 얘기들을 모두 회의록에 정리해서 공유하면 좋을 것 같습니다.
- 변경 사항이 있거나 의문이 있으면 채팅이 아니라 직접 게더에서 상의해서 정하면 좋을 것 같습니다. 채팅으로 한 줄 한 줄씩 치면서 얘기하면 기다리느라 힘이 빠집니다…
- 부득이한 상황이 아니라면 팀회의 참석해서 작업 진행 상황을 부서(?) 별로 공유가 좀 잘되도록 노력했으면 좋겠다.
